binary_search_tree/binary_search_tree.py

```diff
@@ -5,13 +5,47 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    x = self
+    while True:
+      if value == self.value:
```

Comment:
so if the value we want to insert as a new node is equal to the value of the node we are inserting on, we don't end up inserting anything?